src/org/labkey/test/components/labkey/ui/samples/SampleTypeDesigner.java

```diff
@@ -27,13 +27,20 @@
 
     private final WebElement _el;
     private final WebDriver _driver;
+    private boolean _useFinishButton;
 
     public SampleTypeDesigner(WebDriver driver)
     {
         _driver = driver;
         _el = Locator.id("app").findElement(_driver); // Full page component
     }
 
+    public SampleTypeDesigner(WebDriver driver, boolean useFinishButton)
+    {
+        this(driver);
+        _useFinishButton = useFinishButton;
+    }
+
```

Comment:
Did you want to change this one as well to not pass a parameter but put the smarts/logic in the getSaveButton?

Reply:
Bad timing. I checked in this change right after you completed your review. I'll re-request review now that this change is in. Thanks.